src/main/java/com/fryrank/model/Address.java

```diff
@@ -0,0 +1,21 @@
+package com.fryrank.model;
+
+import lombok.Data;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
```

Comment:
I was reading that `@Data` includes the functionality of `@RequiredArgsConstructor`... so this line is not needed.

https://projectlombok.org/features/Data

Reply:
Done in [55c5367](https://github.com/NickPriv/FryRank/pull/10/commits/55c5367ea13efbcf23eacaf6e03235baa4982132).